src/dodal/devices/bimorph_mirrors/CAENels_bimorph_mirror_16_channel.py

```diff
@@ -0,0 +1,174 @@
+    )
+    channel_15_voltage_target_readback_value: EpicsSignalRO = Component(
+        EpicsSignalRO, "C15:VOUT_RBV"
+    )
+    channel_16_voltage_target_readback_value: EpicsSignalRO = Component(
+        EpicsSignalRO, "C16:VOUT_RBV"
+    )
+
+    channel_9_shift: EpicsSignalRO = Component(EpicsSignalRO, "C9:SHIFT")
+    channel_10_shift: EpicsSignalRO = Component(EpicsSignalRO, "C10:SHIFT")
+    channel_11_shift: EpicsSignalRO = Component(EpicsSignalRO, "C11:SHIFT")
+    channel_12_shift: EpicsSignalRO = Component(EpicsSignalRO, "C12:SHIFT")
+    channel_13_shift: EpicsSignalRO = Component(EpicsSignalRO, "C13:SHIFT")
+    channel_14_shift: EpicsSignalRO = Component(EpicsSignalRO, "C14:SHIFT")
+    channel_15_shift: EpicsSignalRO = Component(EpicsSignalRO, "C15:SHIFT")
+    channel_16_shift: EpicsSignalRO = Component(EpicsSignalRO, "C16:SHIFT")
+
+    channel_9_voltage_out: EpicsSignal = Component(EpicsSignal, "C9:VOUT")
+    channel_10_voltage_out: EpicsSignal = Component(EpicsSignal, "C10:VOUT")
+    channel_11_voltage_out: EpicsSignal = Component(EpicsSignal, "C11:VOUT")
+    channel_12_voltage_out: EpicsSignal = Component(EpicsSignal, "C12:VOUT")
+    channel_13_voltage_out: EpicsSignal = Component(EpicsSignal, "C13:VOUT")
+    channel_14_voltage_out: EpicsSignal = Component(EpicsSignal, "C14:VOUT")
+    channel_15_voltage_out: EpicsSignal = Component(EpicsSignal, "C15:VOUT")
+    channel_16_voltage_out: EpicsSignal = Component(EpicsSignal, "C16:VOUT")
+
+    channel_9_voltage_out_readback_value: EpicsSignal = Component(
+        EpicsSignal, "C9:VOUT_RBV"
+    )
+    channel_10_voltage_out_readback_value: EpicsSignal = Component(
+        EpicsSignal, "C10:VOUT_RBV"
+    )
+    channel_11_voltage_out_readback_value: EpicsSignal = Component(
+        EpicsSignal, "C11:VOUT_RBV"
+    )
+    channel_12_voltage_out_readback_value: EpicsSignal = Component(
+        EpicsSignal, "C12:VOUT_RBV"
+    )
+    channel_13_voltage_out_readback_value: EpicsSignal = Component(
+        EpicsSignal, "C13:VOUT_RBV"
+    )
+    channel_14_voltage_out_readback_value: EpicsSignal = Component(
+        EpicsSignal, "C14:VOUT_RBV"
+    )
+    channel_15_voltage_out_readback_value: EpicsSignal = Component(
+        EpicsSignal, "C15:VOUT_RBV"
+    )
+    channel_16_voltage_out_readback_value: EpicsSignal = Component(
+        EpicsSignal, "C16:VOUT_RBV"
+    )
+
+    channel_9_status: EpicsSignal = Component(EpicsSignal, "C9:STATUS")
+    channel_10_status: EpicsSignal = Component(EpicsSignal, "C10:STATUS")
+    channel_11_status: EpicsSignal = Component(EpicsSignal, "C11:STATUS")
+    channel_12_status: EpicsSignal = Component(EpicsSignal, "C12:STATUS")
+    channel_13_status: EpicsSignal = Component(EpicsSignal, "C13:STATUS")
+    channel_14_status: EpicsSignal = Component(EpicsSignal, "C14:STATUS")
+    channel_15_status: EpicsSignal = Component(EpicsSignal, "C15:STATUS")
+    channel_16_status: EpicsSignal = Component(EpicsSignal, "C16:STATUS")
```

Comment:
Should: Since you started this project, we realised we don't need the type hints as well as using `Component` - the `Component` does it for you. Eg we can just have  `channel_9_status = Component(EpicsSignal, "C9:STATUS")`